node/Cargo.toml

```diff
@@ -389,6 +389,11 @@ branch = "peaq-polkadot-v0.9.29"
 [dependencies.peaq-primitives-xcm]
 path = "../primitives/xcm"
 
+[dependencies.peaq-pallet-storage-rpc]
```

Comment:
After we merge the MR in the peaq-storage-pallet, please change the branch here because we'll delete the feature branch.